### PR TITLE
Commit with changes made for HW #1

### PR DESCRIPTION
**Pull request by dstad418**
**CODE CHANGES I MADE:**
I completed the HW #1 assignment. This included updates to selectors.md, and palindrome.html/.js

**WHY I MADE THESE CHANGES:**
To practice my CSS and to get the palindrome program to work and display correctly. 

**WHAT FEEDBACK I AM LOOKING FOR:**
I am hoping to see if it meets the criteria for 'good code' given the standards we went over today in class.

### CODE REVIEW

**Comment by dstad418** [comment]
Wait, was I supposed to put a blue border around the Palindrome calculator, a la the PDF?
Also, WAVE showed no accessibility issues.

**Review by SayleeMJ — COMMENTED** [review]
Great work! Your code is in line with the Wave and Airbnb guidelines, and I was able to test it without any issues. The only thing to keep in mind is that it doesn't quite meet the CSS requirements outlined in homework1.pdf.

**Comment by dstad418** [comment]
Hey Saylee,
I saw "If there are accessibility issues due to default Bootstrap colors, you will need to update colors using CSS."
I see that WAVE isn't showing any accessibility issues with my implementation. Are you talking about the blue border in the pdf pictures? If so, I was confused on that too, but I didn't see any requirement in the PDF. 
I'll see if Caterina wanted us to include that!

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+      if (value === "") {
+        output.innerHTML = "";
+        return;
+      }
+  
+      // If value is less than 0 OR the value was a non-number (i.e.: text, symbols, etc.)
+      if (value < 0 || isNaN(value)) {
+        output.innerHTML = '<span class="text-danger">Please enter a valid positive number.</span>';
+        return;
+      }
```
Comment (lebenpdx) [inline hw1/02-palindrome.js]: I am struggling to trigger this for conditional for symbols. Instead the output remains blank.

File: hw1/02-palindrome.js [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+    });
+  
+    // Function to check if num is palindrome
+    function isPalindrome(num) {
+      // Turns number to string
+      const strNum = String(num);
+      // Parses string characters to an array, flips array values, rejoins them.
+      // Returns true if the values match, else false
+      return strNum === strNum.split("").reverse().join("");
```
Comment (lebenpdx) [inline hw1/02-palindrome.js]: Rule [19.6](https://github.com/airbnb/javascript#whitespace--chains) states that we should break this method chain up with indentation when it is longer than 2 methods.

**Review by lebenpdx — COMMENTED** [review]
I have finished my code review. I also was wondering about the blue border, since there was nothing explicitly asking for it besides it being shown in the image.

**Comment by mattsloe** [comment]
Great work on this one. I appreciate Ben's analysis of the code for the AirBnB guidelines (I'm still getting used to the guidelines)

**Comment by dstad418** [comment]
Made changes and confirmed CSS requirements.

**Pull request merged**